src/hotspot/share/utilities/globalDefinitions_visCPP.hpp

```diff
@@ -32,19 +32,19 @@
 // declarations and a few frequently used utility functions.
 
 # include <ctype.h>
-# include <string.h>
 # include <stdarg.h>
-# include <stdlib.h>
 # include <stdint.h>
+# include <stdlib.h>
+# include <string.h>
 # include <stddef.h>// for offsetof
 # include <sys/stat.h>
 # include <io.h>    // for stream.cpp
 # include <float.h> // for _isnan
 # include <stdio.h> // for va_list
```

Comment:
These aren't being flagged as out of order by the checker, because of the trailing comments.
And the comment for `<stdio.h>` is odd, since that's already being brought in by `<stdarg.h>`.

Reply:
> And the comment for <stdio.h> is odd, since that's already being brought in by <stdarg.h>.

I see @kimbarrett, fixed in 8ebbfaf61823933895ab3757d352eaa90051fde8. I just removed the comment